tests/test_fakes.py

```diff
@@ -798,7 +799,10 @@ def test_info_schema_databases(conn: snowflake.connector.SnowflakeConnection):
         cur.execute("create database db2")
         cur.execute("select * from information_schema.databases")
 
-        assert cur.fetchall() == [
+        databases = typing.cast(list[dict], cur.fetchall())
+        result = [d for d in databases if d["database_name"] != "_fs_global"]
```

Comment:
Can we filter this out in the implementation, see [SQL_CREATE_INFORMATION_SCHEMA_DATABASES_VIEW ](https://github.com/tekumara/fakesnow/blob/b8e38a6ecf28173c11713a8ab5ca6287c8114ebb/fakesnow/info_schema.py#L64), rather than expose this implementation detail to end users?